src/ProjectDataProvider.ts

```diff
@@ -330,11 +330,11 @@ export module ProjectDataProvider {
         `-u`,
         `github.com/fabric8-analytics/cli-tools/gomanifest`,
         `&&`,
-        Config.getGoExecutable(),
-        `run`,
-        `github.com/fabric8-analytics/cli-tools/gomanifest`,
-        `"${vscodeRootpath}"`,
-        `"${goGraphFilePath}"`,
+        // running gomanifest directly for Go 1.16 support
+        `gomanifest`,
```

Comment:
This does not work on Mac OS, I think system will not find `gomanifest` if GOROOT or GOPATH is not set properly. Need to have generic command that can work for all use-case.

Reply:
Thanks for checking on mac

Reply:
I have made generic changes to set GOPATH to temporary directory and use `gomanifest` with absolute path to temp directory. This changes are merged to master code. Can you please check if this resolves the issue your were facing? You can take latest extension VSIX from https://github.com/fabric8-analytics/fabric8-analytics-vscode-extension/suites/2525745446/artifacts/54726285

Reply:
Thank you very much. It worked perfectly on Linux, but it breaks on Windows. I then closed this PR and made PR #510 to fix it for Windows.